test/integration/smart_answer_flows/uk_benefits_abroad_test.rb

```diff
@@ -198,48 +198,32 @@ class UKBenefitsAbroadTest < ActiveSupport::TestCase
         end
       end
 
-      context "answer Jersey or Guernsey" do
-        setup do
+      context "answer Jersey" do
+        should "go to Channel Islands outcome" do
           add_response "jersey"
+          assert_current_node :jsa_not_entitled_outcome
         end
-        should "ask you how long are you going abroad for?" do
-          assert_current_node :how_long_abroad?
-        end
+      end
 
-        context "answer less than one year" do
-          should "go to Channel Islands outcome" do
-            add_response "one_year_or_less"
-            assert_current_node :jsa_channel_islands_outcome
-          end
+      context "answer Guernsey" do
+        should "go to JSA SS going abroad outcome" do
+          add_response "guernsey"
+          assert_current_node :jsa_social_security_going_abroad_outcome
         end
+      end
 
-        context "answer more than one year" do
-          should "go to JSA SS going abroad outcome" do
-            add_response "more_than_one_year"
-            assert_current_node :jsa_social_security_going_abroad_outcome
-          end
+      context "answer Gibraltar" do
+        should "go to JSA EEA going abroad outcome" do
+          add_response "gibraltar"
+          assert_current_node :jsa_eea_going_abroad_maybe_outcome
         end
       end
 
       context "answer Kosovo" do
-        setup do
+        should "go to JSA SS going abroad outcome" do
+          # add_response "one_year_or_less"
           add_response "kosovo"
-        end
-        should "ask you how long are you going abroad for?" do
-          assert_current_node :how_long_abroad?
```

Comment:
Should the [`how_long_abroad?` template](https://github.com/alphagov/smart-answers/blob/3cb8aefe251a7fcf0bef3f2d778d5f0e44ae34e5/lib/smart_answer_flows/uk-benefits-abroad/questions/how_long_abroad.erb#L1-L8) be deleted too?